woodwork/logical_types.py

```diff
@@ -64,6 +64,9 @@ def transform(self, series):
                 raise TypeConversionError(series, new_dtype, type(self))
         return series
 
+    def validate(self, series):
+        return
```

Comment:
Not sure if it matters to use `return` or `pass` here

Reply:
raise a `NotImplementedError`?

Reply:
Do we want to require every logical type to implement a validate function? I guess it would make sense to at least confirm that every logical type has been converted to the proper dtype as a result of the transform -- unless that's already being done elsewhere.

Reply:
Oh, right. No I don't (at this time).